Makefile

```
@@ -20,7 +20,7 @@ build: clean deps
 	@# Install OpenFisca-Extension-Template for deployment and publishing.
 	@# `make build` allows us to be be sure tests are run against the packaged version
 	@# of OpenFisca-Extension-Template, the same we put in the hands of users and reusers.
-	python setup.py bdist_wheel
+	python -m build
```

And ensure that no other wheel is installed as in [openfisca-core](https://github.com/openfisca/openfisca-core/blob/752c3befde4ee2d5acb36cf5ae9a35ea2e9f976a/openfisca_tasks/publish.mk#L10)?
```suggestion
	python -m build
	pip uninstall --yes openfisca-extension-template
```